bundles/af-core/pom.xml

```diff
@@ -86,9 +86,15 @@
                         <!-- Enable processing of OSGI metatype annotations -->
                         <_metatypeannotations>*</_metatypeannotations>
                         <Import-Package>
-                            javax.annotation;version=0.0.0,
+<!--                            javax.annotation;version=0.0.0,-->
+                            <!-- https://github.com/jhy/jsoup/issues/1616 -->
+                            javax.annotation;version=!;resolution:=optional,javax.annotation.meta;version=!;resolution:=optional,
                             *
                         </Import-Package>
+                        <Embed-Dependency>
+                            core.wcm.components.core;
```

Comment:
why jsoup is embedded?

Reply:
it is coming from core.wcm.components.core;

Reply:
com.adobe.aem:core-forms-components-examples-core:bundle:1.1.7-SNAPSHOT
[INFO] +- com.adobe.cq:core.wcm.components.core:jar:2.21.2:provided
[INFO] |  +- org.apache.sling:org.apache.sling.servlets.annotations:jar:1.2.4:provided
[INFO] |  \- org.jsoup:jsoup:jar:1.14.3:provided
[INFO] +- org.jetbrains:annotations:jar:17.0.0:provided
[INFO] +- uk.org.lidalia:slf4j-test:jar:1.2.0:test
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.6:provided

Reply:
@nit23uec I have tested the newly created env and DoR is working fine

Reply:
ideally, we wcm core components should bring their own jsoup. I'm not blocking the merge due to this. But this is undesired.